### ISSUE TITLE
[Top Bar in Foundation Client App] JavaScript does not appear to be running.

### ISSUE THREAD

**Issue by ansorensen** [zurb/foundation-sites#8441]
**How can we reproduce this bug?**
1. Create a new project with Foundation client (libsass or compass, both behave the same)
2. Copy in the 'Responsive Toggle' html into the file

**What did you expect to happen?**
Top bar displayed horizontally

**What happened instead?**
Top bar appears broken, spilling vertically down the page, with items that should be hidden except on mobile still visible.  Looks like JS isn't updating the top bar HTML, even though there are no console errors and $(document).foundation() is being run.

![broken_topbar2](https://cloud.githubusercontent.com/assets/2054041/13937132/d3e40e06-ef7f-11e5-9760-805585cd9e4f.png)

Best guess on the root problem: Foundation client uses 'bower-foundation', which according to [this Git repo](https://github.com/zurb/bower-foundation) only goes up to version 5.  It's out of date, and needs to be configured to grab 'foundation-sties' from [this repo](https://github.com/zurb/foundation-sites) instead.

**Comment by colin-marshall** [comment]
@ansorensen `foundation-sites` is the new name for the bower package as of version 6. So run `bower install foundation-sites` from here on out to get version 6+.

**Comment by ansorensen** [comment]
Yes, I pointed that out in my post, last paragraph.  The point of the post is to point out that Foundation's own command line client (the use of which is encouraged in docs and tutorials) uses the old package, not the new one.  It's confusing to someone trying to get started with Foundation.  It should be updated.

**Comment by colin-marshall** [comment]
Sorry I misunderstood you. You were saying Foundation _Client_ and I believe you meant Foundation **CLI** (Command Line Interface).

What version of the CLI are you running? You probably just need to update the CLI. Run `foundation -v` to get the version. Thanks.

**Comment by ansorensen** [comment]
Sorry about the terminology.  I installed the client this weekend (so I would hope it would be the latest), and just ran 'npm update -g foundation-cli', and there are two issues:

1) no 'foundation -v' command
2) 'foundation  version' outputs v1.0.4

So maybe your NPM repo is out of date?

**Comment by colin-marshall** [comment]
I believe the problem is on your end as the version you are using is 1.5 years old. Run `npm uninstall -g foundation-cli`, then run `npm cache clean`, then run `npm install -g foundation-cli` and see what that does for you.

If that doesn't work I suspect you have an older version of the CLI installed somewhere that is getting loaded in your path before the current version.

**Comment by ansorensen** [comment]
That did work, although it's a little strange that I should have a version that's 1.5 years old when I installed it for the first time this weekend. I'll see if I can hunt through my command line history and retrace my steps, and see if I was working from out of date docs.

**Comment by ansorensen** [comment]
Thanks for helping me sort this out. As someone new to the framework, it's been a little confusing to figure out where to start.  I originally installed the foundation-cli via gem install, as directed by the first tutorial video on your tutorials page, ('Installing Sass with the Foundation CLI').  Had I found http://foundation.zurb.com/sites/docs/installation.html, I would have been directed to the npm install command, but it's much quicker to get to the tutorials page than to actually hunt down where to download the sass version:

Foundation Home > Learn Tutorials > Tutorials (first video on the page) vs.
Foundation Home > Install via Sass > Install foundation for Sites > Installation (npm command is 'below the fold' on my mac)

I'd recommend updating the tutorials, and also making sure that your site architecture makes it easier to find installation instructions.  Not being able to find the up-to-date instructions is why I ended up with an out of date client in the first place.

I know Github is generally for bugs, is there a better place for this feedback?  I'd like to save other new-to-Foundation developers the same headache.
